content/blue_green_deployments/tabs/deployment_and_testing.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Embedded tab content"
```

Comment:
Could you also add a section on deploying a bad image that fails? It would be great to showcase the rollback functionality.

Reply:
Added automated rollback section